src/wasm-type.h

```diff
@@ -58,6 +58,15 @@ struct Rtt;
 enum Nullability { NonNullable, Nullable };
 enum Mutability { Immutable, Mutable };
 
+enum Signedness {
+  // The value has signedness, and is signed, like e.g. int32_t.
+  Signed,
+  // The value has signedness, and is unsigned, like e.g. uint32_t.
+  Unsigned,
+  // The value has no concept of signedness, like e.g. float or raw bits.
+  Nonsigned
```

Comment:
I would prefer `None` here to visually differentiate more from the other two options.

Reply:
I see your point, but it would have to be an enum class, then, unlike the others, to force `Signedness::None`. But ` Signedness::Signed` seems burdensome, hmm...

Perhaps `Sign::Signed/Unsigned/None`? Also kind of weird though...

Reply:
Oh, good point about the enum class being unwieldy. How about `NoSign`, then?

Reply:
`NoSign` sounds good!